src/sections/create-dataset/MetadataBlockFormFields/MetadataFormField/Fields/EmailField.tsx

```diff
@@ -0,0 +1,25 @@
+import { Form } from '@iqss/dataverse-design-system'
+import { forwardRef } from 'react'
+
+interface Props {
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void
+  isInvalid: boolean
+  placeholder: string
+}
+
+export const EmailField = forwardRef(function EmailField(
```

Comment:
They have labels, certainly, I was just trying to assert a specific type of field was represented.
But I remember I wanted to simplify all those fields because they were really identical, so now I removed all identical component, and just use `<Form.Group.Input />` or `<Form.Group.TextArea />` for all fields, except vocabulary fields of course and I add a `data-fieldtype` to assert the type in tests.
```
cy.findByLabelText(/Depth Coverage/)
  .should('exist')
  .should('have.data', 'fieldtype', TypeMetadataFieldOptions.Float)
```